getPeerScript.js

```diff
@@ -0,0 +1,11 @@
+// returns a yarn command to re-add peer dependencies which as a side effect installs them into node_modules
+const deps = require('./package').peerDependencies
+const res = []
+for (let dep in deps) {
+  res.push(`"${dep}@${deps[dep]}"`)
+}
+
+const depStr = res.join(' ')
+if (depStr.length) {
+  process.stdout.write(`yarn add --peer --no-lockfile ${depStr}`)
```

Comment:
I'm not sure we want to  force the usage of yarn, if this script gets used by users of this package as well. I would rather see this returned as a string, so that the users of the script could do something like:
```
npm install `node ./getPeerScript.js`
```
(i'm not sure about the flags to define as peer for npm) or if they use yarn:
```
yarn add --peer --no-lockfile `node ./getPeerScript.js`
```

Reply:
This script was not intended for users of this package. It's just for developers of this package and for running the tests on CI.
It does not modify your `package.json` or lock file.

Reply:
I get that, but you would still need yarn as peer dependency

Reply:
I can either add a detection whether you use `npm` or `yarn` or use `$npm_execpath` but there was no intention to have this work in any environment.
All I need from this script is to automatically install peer dependencies on CI.
I'll however remove `yarn eslint` from the `test` script so that we don't combine both `npm` and `yarn` in `package.json`.